pkg/etl/Kconfig

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 Jens Wetterich <jens@wetterich-net.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_ETL
+    bool "Embedded Template Library"
+    depends on TEST_KCONFIG
```

Murdock error:
> /tmp/dwq.0.8167323357704165/4d8b5b6a5b794425a415c79057f65e0a/Makefile.base:80: *** Found C++ source, but "cpp" module is not used. Enable the MODULE_CPP in Kconfig.  Stop.
```suggestion
    depends on MODULE_CPP
    depends on TEST_KCONFIG
```
Do I need to add this here?

---

let's try it :wink:

---

Seems to be wrong.
```
[genconfig.py]:ERROR-=> The parameter PACKAGE_ETL could not be set to y.
[genconfig.py]:ERROR-   Check the following unmet dependencies: MODULE_CPP (=n)
```
Could anyone with Kconfig knowledge give a hint? Thanks! Maybe @MrKevinWeiss who had a good hint on this before?

---

Just do what `genconfig.py` is telling you: It complains that `MODULE_CPP` is set to `n`, so set it to `y` ;-).

---

(Kconfig does not pull in dependencies automatically, it will just warn you about it when you did not select the module).